src/commands/package/version/report.ts

```diff
@@ -191,10 +191,12 @@ export class PackageVersionReportCommand extends SfCommand<PackageVersionReportR
     });
     if (!verbose) {
       displayRecords.splice(displayRecords.map((e) => e.key).indexOf('Id'), 1);
-      displayRecords.splice(
-        displayRecords.map((e) => e.key).indexOf(pvlMessages.getMessage('convertedFromVersionId')),
-        1
-      );
+      if (record.ConvertedFromVersionId === null || record.ConvertedFromVersionId === ' ') {
```

Comment:
```suggestion
      if (!record.ConvertedFromVersionId?.trim()) {
```
This will cause the condition to be entered with values of: null, undefined, empty string, only space characters, and zero.